AEXS/aex-draft_data_serialization.md

```diff
@@ -1,159 +0,0 @@
-# AEX 7
```

Comment:
This change also has nothing to do here.

Reply:
Could you please add this draft again? If the author wants to withdraw it then it is up to them to do so.

Reply:
This file is a duplicate with aex-7.md`. It seams that author (@davidyuk ) forgot to remove it.

Reply:
Please confirm if you want to keep the duplicate.

Reply:
@knarz  - could you confirm if you want to keep the duplicate? I'm away for next 2 weeks and it would be great to close this topic today.

Reply:
Hey, thank you for the PR. I'll merge it as is

Reply:
> This file is a duplicate with `aex-7.md`. It seams that author (@davidyuk ) forgot to remove it.

I'm not an author of this draft, @AndreasGassmann created it in #15